thirdeye/thirdeye-frontend/tests/integration/pods/components/range-pill-selectors/component-test.js

```diff
@@ -85,15 +93,16 @@ module('Integration | Component | range pill selectors', function(hooks) {
       'Label of 3nd pill is correct');
     assert.equal(
       $rangeInput.val(),
-      `${ACTIVE_RANGE_START} - ${ACTIVE_RANGE_END}`,
-      'Date range is accurate');
+      `${moment(START_DATE).startOf('day').format(UI_DATE_FORMAT)} - ${moment(END_DATE).endOf('day').format(UI_DATE_FORMAT)}`,
+      'Date range displayed in date-range-picker input is accurate');
 
     // Clicking to activate date-range-picker modal
-    await click($rangeInput);
+    await run(() => $rangeInput.click());
```

Comment:
why is `run` needed here? I believe we can simply use the `click` helper here

```
await click('.daterangepicker-input')
```

Reply:
Ember component testing docs recommend using it to prevent unintended behavior: https://guides.emberjs.com/v2.18.0/testing/testing-components/#toc_testing-user-interaction

`We recommend using native DOM events wrapped inside the run loop or the ember-native-dom-helpers addon to simulate user interaction and test that the title is updated when the button is clicked.
Using jQuery to simulate user click events might lead to unexpected test results as the action can potentially be called twice.`